nginx-overrides.conf

```diff
@@ -0,0 +1,5 @@
+server {
+  port_in_redirect off;
+  listen 4000;
+  error_page 403 404 /404.html;
```

Comment:
Can you check my comments on https://github.com/docker/docker.github.io/pull/5351 w.r.t. the configuration file, and the target directory for the html (`/usr/share/nginx/html`)?